src/status_im2/contexts/chat/messages/navigation/style.cljs

```diff
@@ -16,13 +16,13 @@
     :background-color (colors/theme-colors colors/white-opa-40 colors/neutral-80-opa-40)}
    position))
 
-(def background-view
+(defn background-view []
   {:position         :absolute
    :top              0
    :left             0
    :right            0
    :height           navigation-bar-height
-   :background-color (colors/theme-colors colors/white-opa-70 :transparent)
+   :background-color (colors/theme-colors colors/white-opa-70 colors/neutral-100-opa-70)
```

Comment:
while you're making changes here can you update this component to use `quo2.theme` with-theme HOC and then pass in `theme` as a 3rd param. e,g

```
(defn background-view [theme]
  {:position         :absolute
   :top              0
   :left             0
   :right            0
   :height           navigation-bar-height
   :background-color (colors/theme-colors colors/white-opa-70 colors/neutral-100-opa-70 theme)
   ```